tests/test_store.py

```diff
@@ -78,5 +78,46 @@ def test_insert(rdf_store, example_graphs):
         assert tuple(f"https://example.org/{part}#{i}" for part in ["subject", "predicate", "object"]) in results
 
 
+@pytest.mark.usefixtures("rdf_store", "example_graphs")
+def test_insert_large(rdf_store, example_graphs):
+    assert rdf_store is not None, "can't perform test without target store"
+
+    # Read large file
+    lg = Graph().parse("./tests/input/AffiliationInfo.ttl", format='turtle')
+    num_triples = len(lg)
+    log.debug(f"{num_triples}")
+
+    # Call the insert method
+    rdf_store.insert(lg, "<urn:test:large-turtle-test>")
+
+    # Verify that the triples are parsed correctly
+    sparql = "SELECT ?subject ?predicate ?object WHERE { ?subject ?predicate ?object }"
```

Comment:
this will yield unexpected side-effects imho (surely when testing vs a local graphdb) 

selection should be narrowed within the named_graph to make sure the number of triples will match (on line 97)